ui/css/perf.css

```diff
@@ -455,3 +455,29 @@ td.alert-confidence {
 .alert-strike {
     text-decoration: line-through;
 }
+
```

Comment:
Could you move these new css directives to a file called `treeherder-loading-overlay.css` and have both perf.html and index.html import it? You can then remove the duplicated css logic from `treeherder-info-panel.css`.